askforhelp.rst

```diff
@@ -63,17 +63,16 @@ The big picture - An opening sentence should state the general problem that you
 
 **A snapshot of your environment** - For Plone and for other relevant products: provide version numbers. e.g., "I'm running Plone 5.0.4 under Python 2.7.11."
 
-**Steps to reproduce the issue** - Give information about your ideas of how this error appeared, what caused it or anything that could lead to reproducing the error on another computer. The expected result .
+**Steps to reproduce the issue** - Give information about your ideas of how this error appeared, what caused it or anything that could lead to reproducing the error on another computer, including your buildout.cfg, versions.cfg, version numbers of installed add-ons, detailed command lines, the complete error message stack. Mention your expected result.
 
-Asking help in real time chat (IRC)
------------------------------------
+Asking for help in online chat
+------------------------------
 
-Plone community real time chat is at the  `#plone <irc://irc.freenode.net/#plone>`_ IRC channel on the freenode IRC network.
+The Plone community uses `online chat <https://plone.org/support/chat>`_, specifically `Gitter <https://gitter.im/plone/public>`_ and traditional Plone IRC chat at the  `#plone <irc://irc.freenode.net/#plone>`_ IRC channel on the freenode IRC network.
 
```

Comment:
Would you mind to point me to the place where "we" made the decision? It seems I missed that, I only can find postings on community where people discus that.

Besides from my own opinion why this is *really bad* I will try to explain it in a hopefully different way, sorry that is getting 'longish' now :).

Please try to think not as Kim the developer but in place of that as Kim the webmaster who is new to Plone and never used it before.
So you are a webmaster, you do not have programmer knowledge, you do not write code, your work environment is your browser and Libre Office.

You have a question about a setting in the CMF, you can't find the answer in the docs (bad bad) but hey there is this chat mentioned on docs.plone.org/askforhelp. Hopefully there people will able to answer me my question, yeah !
Your click on the link and you end up on the Gitter page, nice, but uh eh I can see there is a chat and I can see people talking but I can not ask a question, first I have to login ...

Ok lets click on sign in ... uh they are asking for a GitHub or Twitter login .. What is GitHub never heard about it, lest check oh for developer and nerds, ok do not need nor have that .... Twitter? Hm .. I will not use my personally account to login for work related stuff, and I am not allowed to use a company account ....

So hey, looks nice but I am not able to use that in anyway ...

Thanks for staying with me !

What I am trying to say, Gitter maybe handy for you and I can see the point of have all the nice GitHub hooks, but I doubt that is that useful for 'normal' people.
So IMHO you make it harder for people to ask for help. If you are a 'developer',a person who has already a GitHub account, yes it maybe helpful, great but for others not at all, you may even push them away, which is bad for Plone.

That is exactly the reason why we thought about this deep and long for docs.plone.org and went with a really 'simple' widget, which people can click and just can write us or attach pictures, no need for a login somewhere.
Again ... if you want to be serious about helping people you also should make it easy and in a way that it is working for them, this may involve slightly more work you you, but like I said it is not about you it is about user.
Going further this is also about marketing, if you make it hard for people you will loose them to other Solutions and you will get knows as hard to use and impossibly to get help.

If I went for all stuff what and how I would like to do docs.plone.org they would be completely different today, I did not do that because it would be 'harder' for people to contribute, so I did not do that, yet ......(you never know), hehe
I you feel that you get 'bananas' the way how IRC works and people disconnect, maybe the discussion should be more 'do we want a chat as support' at all ?

So in short I am serious concerned and against merging that in the current state. If this would get re-written in a way, where it is better and more explained that Gitter requires to login and also the reason why 'we' choose for that, I may fine, but sorry not how it is now.

@tkimnguyen please do not take it personally, this is **not meant** to insult you at all ! I am trying to make it better for less technical people because I am concerned that we 'forget' them again and I am concerned about the way Plone is moving .....